### ISSUE TITLE
Feature: Add parameter text to clipboard

### ISSUE THREAD

**Issue by edakiri** [astrand/xclip#51]
This is a feature request to
add the text of a parameter or parameters to the clipboard or selection. The text of the parameter itself would be added -- not a file or input identified by it.

**Comment by hackerb9** [comment]
I agree that sounds like a good idea and should be easy to implement. I don't know if the author of `xclip` would accept it, though. 

In the meantime, here's an ugly workaround using Bash's "Here Strings":

```
xclip <<< "I am not a fish."
```
